charts/kyverno/values.yaml

```diff
@@ -49,22 +49,26 @@ podSecurityContext: {}
 priorityClassName: ""
 
 antiAffinity:
-  # This can be disabled in a 1 node cluster.
-  enable: true
-
-affinity:
-  podAntiAffinity:
-    preferredDuringSchedulingIgnoredDuringExecution:
+  preferredDuringSchedulingIgnoredDuringExecution:
     - weight: 1
       podAffinityTerm:
         labelSelector:
           matchExpressions:
-          - key: app.kubernetes.io/name
-            operator: In
-            values:
-            - kyverno
+            - key: app.kubernetes.io/name
+              operator: In
+              values:
+                - kyverno
         topologyKey: kubernetes.io/hostname
 
+podAffinity: {}
+
+nodeAffinity: {}
+
+affinity:
+  podAntiAffinity:
```

Comment:
If the key for the actual antiAffinity is `antiAffinity` then either this should be `antiAffinity` or change the `antiAffinity` to `podAntiAffinity`.  I realize now that trying to keep same key doesn't really matter since the structure is changing so going to be breaking change no matter what

Reply:
I got an solution for this! 
maybe not the a 100% clear, but it won't be a (big) braking change. just if someone overwrite the `podAntiAffinity`

The Toggle stay the same. I don't even know why I didn't get this in my first commit. 

Thank you for you time and the review. Hopefully ok now.